src/components/Phone.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { inputPhoneActionCreator } from '../redux/actionCreators';
+
+function Phone(props) {
```

Comment:
it looks like copy/paste, I see 3 very similar controls. Please make it with a common class to avoid of copying the code